local.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:20.04
```

Comment:
This looks identical to the one in tests/host: https://github.com/haydnv/tinychain/blob/main/tests/apps/Dockerfile

Is there any difference?